mantidproject/mantid
# LoadSassena stopped sorting by Qvectors

Simple correction to change an iterator in-place.

**To test:**
Download file [signal.h5.zip](https://github.com/mantidproject/mantid/files/1126101/signal.h5.zip) to `/tmp` then unzip and run:

`LoadSassena(Filename="/tmp/signal.h5", OutputWorkspace="signal")`

Then verify that the **first column** of workspace `signal_qvectors` does increase from 0 to 0.5 inverse Angstroms. That's all!

Fixes #19968

*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
